cmd/kubelet/app/server.go

```diff
@@ -713,6 +713,8 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 		}
 	}
 
+	klog.InfoS("Using cgroup driver", "cgroupDriver", s.CgroupDriver)
```

Comment:
is `KubeletCgroupDriverFromCRI` enabled, this will be double-logged. Maybe put it in `else` statement or remove it from https://github.com/kubernetes/kubernetes/blob/a7702cb47babcff60518b510293dbb41ba8eab35/cmd/kubelet/app/server.go#L1431

Reply:
Another option would be bumping to `V(4)` or so. It was added mostly for debugging, it will indeed possibly double logged but exactly once, which should not be a big deal.
But also removing this line entirely is not a big deal at all.

Reply:
it seems to me the log is still present in the last upload, can we please check again?

Reply:
whoops, rebased wrong